src/tls/cert.h

```diff
@@ -60,9 +60,8 @@ namespace tls
       {
         // Peer hostname is only checked against peer certificate (SAN
         // extension) if it is set. This lets us connect to peers that present
-        // certificates with IPAddress in SAN field (mbedtls does not parse
-        // IPAddress in SAN field). This is OK since we check for peer CA
-        // endorsement.
+        // certificates with IPAddress in SAN field. This is OK since we check
```

Comment:
Oh, that's an interesting one! I've created https://github.com/microsoft/CCF/pull/3431 to add further verification in the join protocol TLS handshake.

Reply:
Should this comment change in that PR, too?

Reply:
I think the comment as you've re-phrased it still holds.